docs/crio.8.md

```diff
@@ -313,6 +313,18 @@ it later with **--config**. Global options will modify the output.
 
 **--default**: Output the default configuration (without taking into account any configuration options).
 
+**--migrate-defaults, -m**="": Migrate the default config from a certain version.
+    To run a config migration, just select the input config via the global
+    '--config,-c' command line argument, for example:
+    ```
+    crio -c /etc/crio/crio.conf.d/00-default.conf config -m 1.17
+    ```
+    The migration will print converted configuration options to stderr and will
+    output the resulting configuration to stdout.
+    Please note that the migration will overwrite any fields that have changed
+    defaults between versions. To save a custom configuration change, it should
+    be in a drop-in configuration file instead. (default: 1.17)
+
```

Comment:
It might be nice to have an option that lists the valid values or point the user to a file where they could see the valid values.  Not such an issue now, but 5 or 6 versions from now in the future.  Something to consider, not necessary for this PR.

Reply:
I added the possible values so that we do not for get it in the future.